user_manual_de/files/access_webdav.rst

```diff
@@ -2,140 +2,137 @@
+    webdav://example.com/nextcloud/remote.php/dav/files/USERNAME/
 
 .. image:: ../images/webdav_dolphin.png
-   :alt: WebDAV Konfiguration mit dem Dolphin Dateimanager
+   :alt: WebDAV-Konfiguration mit dem Dolphin-Dateimanager
 
 Sie können einen permanenten Link zu Ihrem Nextcloud-Server erstellen:
 
 #. Öffnen Sie Dolphin und klicken Sie auf "Netzwerk" im linken Fensterbereich
    unter "Persönliche Ordner".
 #. Klicken Sie auf das Symbol mit der Aufschrift **Netzwerkordner hinzufügen**.
-   Ein Fenster sollte bereits mit der Auswahl WebDAV erscheinen.
 #. Wenn WebDAV nicht ausgewählt ist, wählen Sie es aus.
 #. Klicken Sie auf **Weiter**.
 #. Geben Sie die folgenden Einstellungen ein:
    * Name: Der Name, den Sie im Bereich **Persönliche Ordner** als Lesezeichen
-     sehen möchten, zum Beispiel Nextcloud.
+     sehen möchten, z.B. Nextcloud.
    * Benutzer: Der Nextcloud-Benutzername, mit dem Sie sich angemeldet haben,
-     zum Beispiel admin.
-   * Server: Der Nextcloud-Domänenname, zum Beispiel **beispiel.de** (ohne
-     **http://** davor oder den weiterführenden Verzeichnisse).
-   * Ordner -- Geben Sie den Pfad ein
-     ``nextcloud/remote.php/dav/files/BENUTZERNAME/``.
+     z.B. "admin".
+   * Server: Der Nextcloud-Domain, z.B. **example.com** (ohne
+     **http://** davor oder den weiterführenden Ordner).
+   * Ordner -- Geben Sie diesen Pfad ein:
+     ``nextcloud/remote.php/dav/files/USERNAME/``.
 #. (Optional) Aktivieren Sie das Kontrollkästchen "Symbol für diesen
    Netzwerkordner anlegen", damit ein Lesezeichen in der Spalte "Persönliche
    Ordner" angelegt wird.
 #. (Optional) Aktivieren Sie das Kontrollkästchen "Verschlüsselung verwenden",
-   um HTTPS zu aktivieren.
+   um HTTPS zu verwenden.
 
 Erstellen von WebDAV-Verbindungen im Linux Terminal
 ---------------------------------------------------
 
 Sie können WebDAV-Verbindungen über ein Linux Terminal erstellen. Dies ist
-nützlich, wenn Sie auf Nextcloud genauso wie auf alle anderen entfernten
-Dateisysteme zugreifen möchten. Im folgenden Beispiel wird gezeigt, wie Sie eine
-Verbindung herstellen und diese bei jedem Systemneustart automatisch auf
-Ihrem Linux-Computer aufgebaut wird.
+nützlich, wenn Sie auf Nextcloud genauso wie auf allen anderen entfernten
+Dateisysteme zugreifen möchten. Im folgenden Beispiel wird gezeigt, wie Sie
+eine Verbindung herstellen und diese bei jedem Systemneustart automatisch auf
+Ihrem Linux-Computer eingebunden wird.
 
-1. Installieren Sie zunächst für das WebDAV Dateisystem den Treiber ``davfs2``.
+1. Installieren Sie zunächst für das WebDAV-Dateisystem den Treiber ``davfs2``.
    Dieser Treiber erlaubt Ihnen Verbindungen zu WebDAV-Freigaben herzustellen.
    Folgende Anweisung installiert den Treiber unter Debian/Ubuntu::
 
     apt-get install davfs2
 
-.. TODO: das komma nach fedora ist zuviel
-2. Für CentOS, Fedora und openSUSE gilt folgendes Kommando::
+2. Für CentOS, Fedora und openSUSE wird Folgendes Kommando verwendet::
```

Comment:
In diesem Fall schreibt man `folgendes` klein, da es sich auf das Kommando bezieht ;)